packages/generate-component-docs/src/__snapshots__/contract.test.ts.snap

```diff
@@ -8214,7 +8214,7 @@ exports[`filterSuggestions 1`] = `
   props: {
     at: (index: number) => FilterableSuggestion<Value> | FilterableGroupedSuggestions<Value> | undefined
     concat: { (...items: ConcatArray<FilterableSuggestion<Value> | FilterableGroupedSuggestions<Value>>[]): (FilterableSuggestion<...> | FilterableGroupedSuggestions<...>)[]; (...items: (FilterableSuggestion<...> | ... 1 more ... | ConcatArray<...>)[]): (FilterableSuggestion<...> | FilterableGroupedSuggestions<...>)[]; }
-    copyWithin: (target: number, start?: number | undefined, end?: number | undefined) => FilterableSuggestions<Value>
+    copyWithin: (target: number, start: number, end?: number | undefined) => FilterableSuggestions<Value>
```

Comment:
`filterSuggestions`'s argument is an array. TypeScript changed, so the type definition for `Array` must have changed as well.

This is a problem with how we extract type information from component exports.